epub33/rs/index.html

```diff
@@ -957,20 +957,18 @@ <h4>Security Considerations</h4>
 
 					<ul>
 						<li>
-							<p>Reading Systems need to behave as if a unique domain were allocated to each Content
-								Document, as browser-based security relies heavily on document URLs and domains.
-								Adopting this approach will isolate documents from each other and from other Internet
-								domains, thereby limiting access to external URLs, cookies, DOM storage, etc.</p>
+							<p>Reading Systems need to behave as if a unique origin were allocated to each <a>EPUB
+									Publication</a>. Adopting this approach will isolate publications from each other,
```

Comment:
I wonder whether 'must' is not a better term. I know this is not a normative section, ie, a MUST is not an option, but the usage of the word 'must' may give more emphasis.

Reply:
Yes, no reason not to start fixing some of these now.